client/components/editor/ActivityContent/index.vue

```diff
@@ -56,9 +59,10 @@ export default {
     focusedElement: null
   }),
   computed: {
+    ...mapChannels({ editorChannel: 'editor' }),
     ...mapGetters('repository', ['activities']),
     ...mapGetters('editor', ['collaboratorSelections']),
-    ...mapChannels({ editorChannel: 'editor' }),
+    ...mapGetters('repository/contentElements', ['processedElements']),
```

Comment:
If it is going to be used only in a single place, it might not make sense to put it in a store (it is better to contexualize it to make it more readable)